modules/recon/profiles-profiles/namechk.py

```diff
@@ -1,70 +1,40 @@
 from recon.core.module import BaseModule
 from recon.mixins.threads import ThreadingMixin
-from http.cookiejar import CookieJar
-from lxml.html import fromstring
+
 
 class Module(BaseModule, ThreadingMixin):
 
     meta = {
         'name': 'NameChk.com Username Validator',
-        'author': 'Tim Tomes (@LaNMaSteR53) and thrapt (thrapt@gmail.com)',
+        'author': 'Tim Tomes (@LaNMaSteR53) and thrapt (thrapt@gmail.com) and Ryan Hays (@_ryanhays)',
         'version': '1.0',
-        'description': 'Leverages NameChk.com to validate the existance of usernames on specific web sites and updates the \'profiles\' table with the results.',
+        'description': 'Leverages NameChk.com API to validate the existance of usernames on specific web sites and '
+                       'pdates the \'profiles\' table with the results.',
+        'required_keys': ['namechk_api'],
         'comments': (
             'Note: The global timeout option may need to be increased to support slower sites.',
```

Comment:
Is this still true? If not, remove the entire comments section.

Reply:
Fixed